### DISCUSSION TITLE
Title break-off / wrapping

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#394] [General]
Is there a way to prevent title break-off such as the ones below? I am on a 15 inch laptop and quite a lot of words in the titles don't fit the cards. 

![Screenshot 2021-08-04 11 16 35](https://user-images.githubusercontent.com/87815729/128156591-513133c2-5dc6-439e-bec3-eb5e2672ea20.png)

**Comment by techsmyth** [comment]
Hi Flori, 

Not an easy one from a user interface design point of view - as it is not nice if some have bigger headers and / or different font sizes. Also it would not make sense to restrict the number of characters allowed in a title as the longer title is only an issue here. 

We could structurally make the font size smaller? 

Am open to other suggestions...easy enough to implement whatever we choose. 

Cheers, Neil

**Reply by FloriSpoelstra** [reply]
Yeah I get what you are saying. Is there maybe an option to hyphenate automatically?
If not, then I wouldn't mind the font size to be a tiny bit smaller.
